packages/nimble-components/src/rich-text/viewer/specs/README.md

```diff
@@ -0,0 +1,3 @@
+# Nimble Rich Text Viewer
+
+The spec of this component is added as part of the [`/rich-text/editor/specs/README.md`](../../editor/specs/README.md)
```

Comment:
nit: I think now we can avoid having two spec files, one for editor and another for viewer redirecting to editor. 

Instead can we have one specs folder under `rich-text` containing the actual spec HLD having both editor and viewer. If viewer branches out into different direction on the dependencies or logic (which is less unlikely), then we may need a separate spec with unique content for viewer, removing any redirection links to editor spec as given here.

But I don't have a strong preference on this suggestion. My concern was to only avoid this spec file which just redirects to another spec file. It's good to keep this open for Nimble code owner's advise on the preferred approach to maintain the spec.

Reply:
Looks good! We've also made separate documents for large new features and linked them from the main spec which can be another approach if large features are added in the future.